packages/reaction-accounts/client/templates/inline/inline.html

```diff
@@ -1,6 +1,6 @@
 <template name="loginInline">
   <div class="accounts-dialog accounts-inline">
-    {{#unless currentUser}}
```

Comment:
I believe `currentUser` is a Meteor.accounts template helper,  and we were extending with the guest logic so that the Accounts dropdowns, forms, all had the same behaviour.  are we sure the user accounts dropdown,etc are behaving the same? I'm seeing cases where the logged in user doesn't see the user accounts/profile and dashboard links, etc.